.travis.yml

```diff
@@ -58,5 +58,5 @@ script:
 
 after_success:
   #Generates coverage report:
-  - ./gradlew --stacktrace coverageReport && cp build/reports/jacoco/mockitoCoverage/mockitoCoverage.xml jacoco.xml || echo "Code coverage failed"
```

Comment:
I think having `--stacktrace` here makes it easier to debug on Travis, so let's revert that change?

Reply:
What about enable build scans in order to use build scans to debug problems like this?

Reply:
@TimvdLippe, I agree. I just shorthanded this option to "-s".

@epeee, good idea, I'll add it.